include/mbedtls/mbedtls_config.h

```diff
@@ -522,6 +522,25 @@
  */
 //#define MBEDTLS_AES_FEWER_TABLES
 
+/**
+ * \def MBEDTLS_AES_ONLY_128_BIT_KEY_LENGTH
+ *
+ * Use only 128-bit keys in AES operations to save ROM.
+ *
+ * Uncommenting this macro removes support for AES operations that are using 192
```

Comment:
(minor) it's more natural in English to say "for AES operations that use 192- or 256-bit keys"